src/index.css

```diff
@@ -1,13 +1,183 @@
-body {
+@import url("https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap");
```

Comment:
do you still need this big css file if you're going to be using styled components in the project?

Reply:
I guess not, 😅 . I added styled-normalize and refactored the css.